packages/csv-parser-price/src/map-custom-fields.js

```diff
@@ -118,8 +120,30 @@ export default (function MapCustomFields() {
     }
   }
 
+  /**
+   * Takes a value and returns a CTP money object
+   * @param value String "USD 123" or object {centAmount: '123', currencyCode: 'USD'}
+   * @returns {{ data: CTP price object, error: Error message if any }}
+   */
   function mapMoney(value) {
     const result = {}
+
+    // handle money value provided as an object
+    if (typeof value === 'object' && value.centAmount && value.currencyCode) {
+      const numberKeys = ['centAmount', 'fractionDigits']
```

Comment:
I think we need to add `preciseAmount` here in order to also support `HighPrecisionMoney `

https://docs.commercetools.com/http-api-types#highprecisionmoney

Reply:
Maybe also a test for it so we test both [Money](https://docs.commercetools.com/http-api-types#money) and [HighPrecisionMoney](https://docs.commercetools.com/http-api-types#highprecisionmoney)?

Reply:
Hi @daern91 .. thx for your comment. I added `preciseAmount` together with a test here e29b4ad